modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCHybridFlowIntegrationTest.java

```diff
@@ -151,6 +158,27 @@ public void testHybridFlowWithNonConfiguredResponseType() throws Exception {
         assertNotNull(oauthErrorCode, "OAuth error code is null");
     }
 
+    @Test(groups = "wso2.is",
+            description = "Test hybrid flow with code id_token token response type",
+            dependsOnMethods = "testHybridFlowWithCodeIdTokenResponseType")
+    public void testHybridFlowWithCodeIdTokenTokenResponseType() throws Exception {
+
+        refreshHTTPClient();
+        final String sessionDataKeyConsent =
+                this.sendHybridAuthRequestPost(this.getConfiguredHybridFlowRequestParams("code id_token token"));
```

Comment:
Shall we use a constant for `code id_token token` response type ?

Reply:
fixed with https://github.com/wso2/product-is/pull/23359/commits/536f81647298adb4f6e5e2196d45b6dcbbbe0c68